### ISSUE TITLE
About "play only playlist"

### ISSUE THREAD

**Issue by xsuper9988** [yuliskov/SmartTubeNext#1531]
When i select "play only playlist"
It will always play an extra video after the end that is not part of the playlist

**Comment by lwaelcyrge** [comment]
Yeah, toggling play only playlist videos continuously seem not working.
